egs/wham/ConvTasNet/run.sh

```diff
@@ -36,11 +20,14 @@ mode=min
 nondefault_src=  # If you want to train a network with 3 output streams for example.
 
 # Training
+# note: for batch_size=8 this needs more than 1 regular GPU (around 12-15G RAM)
+#   so id="0,1" should be specified. But of course when users try various
+#   network arch setting below this can change.
```

Comment:
We could reduce the default batch size if needed. 
But adding a comment that larger batch needs larger memory is not useful IMO.